src/Pages/Components/NewProjectModal/NewProjectModal.js

```diff
@@ -1,51 +1,80 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 import { Formik } from "formik";
 import * as yup from "yup";
 import { Form, Row, Col, InputGroup, Input } from "react-bootstrap";
-import { doCreateProject } from "../../../Firebase/firebase";
+import { doCreateProject, doEditProject } from "../../../Firebase/firebase";
 import { useHistory } from "react-router";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faUpload } from "@fortawesome/free-solid-svg-icons";
 import "./NewProjectModal.scss";
 
-const NewProjectForm = ({ onClose }) => {
+const NewProjectForm = ({ onClose, project }) => {
   console.log(onClose);
+  console.log(project);
+  const [image, setImage] = useState('');
+  var initialValue = {};
+  var defaultImage='https://images.unsplash.com/photo-1639413665566-2f75adf7b7ca?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxlZGl0b3JpYWwtZmVlZHwyN3x8fGVufDB8fHx8&auto=format&fit=crop&w=500&q=60';
+  if(project!==undefined){
+  if(project.photo!==undefined){
+  defaultImage=project.photo;}
+    
+     initialValue ={
+      title: project.name,
+      desc: project.desc,
+      links: project.links.toString(),
+      contactNo: project.contactNo,
+      githubLink: project.githubLink,
+      tags: project.tags?project.tags.toString():"",
+      teamMembers: project.teamMembers,
+      photo: project.photo,
+    }
+  }
+  else{
+     initialValue = {
```

Comment:
We could technically, just get rid of the if else and instead use the OR (||) operator to make the code much cleaner I think.
something like: initalValue = {
 title: project.name || '',
 desc: project.desc || '',
...
}